resources/problem-vote.js

```diff
@@ -0,0 +1,91 @@
+var voteChart = null;
+
+function init_vote_form() {
+    $('#delete-vote-form').on('submit', function (e) {
+        e.preventDefault();
+        $.ajax({
+            url: $('#delete-vote-form').prop('action'),
+            type: 'POST',
+            data: $('#delete-vote-form').serialize(),
+            success: function () {
+                $('#vote-button').text(gettext('Vote on problem points'));
+                $.featherlight.close();
+            },
+            error: function (data) {
+                alert(gettext('Unable to delete vote: {error}').replace('{error}', data.responseText));
+            }
+        });
+    });
+
+    $('#vote-form').on('submit', function (e) {
+        e.preventDefault();
+        $.ajax({
+            url: $('#vote-form').prop('action'),
+            type: 'POST',
+            data: $('#vote-form').serialize(),
+            success: function (data) {
+                $('#vote-button').text(gettext('Change vote (%(points)d)').replace('%(points)d', data.points));
```

Comment:
What. Surely `replace` is the wrong thing. I _think_ [interpolate](https://docs.djangoproject.com/en/4.1/topics/i18n/translation/#interpolate) is the right thing.

Reply:
done
`interpolate` only supports `%s` and `%(foo)s`, so i used `%s`